guides/it/laptop-security.md

```diff
@@ -75,10 +75,119 @@ Made Tech do not have a preferred/approved Anti-virus tool for Linux, but it is
 #### ClamAV
 [ClamAV](https://www.clamav.net/) is an open source AV tool that is popular on Linux - it can be configured in a way that meets the requirements above, but this involves enabling on-access scanning.
 
+In can be installed from the Ubuntu repositories:
```

Comment:
I'd deliberately not included instructions, partly because I didn't want us to have to maintain them, but also because they're quite Ubuntu specific and I suspect there are other user distros in use.  But on reflection I think it's a good idea to include them.

Do you think it's worth breaking the AV section into a separate guide, like the the VPN guide?